doc/CHANGELOG.md

```diff
@@ -14,7 +14,7 @@ Syntax: `- short text describing the change _(Your Name)_`
 - Fix wrong placement of plantings _(Paul)_
 - _()_
 - CI: Send email when master fails #109 _(4ydan)_
-- _()_
+- CI: Add link checker to CI pipeline #555 _(4ydan)_
```

Comment:
I didn't see where the check in CI runs, does this PR already add the link check to the CI pipeline?

Reply:
Yes, here.
![image](https://github.com/ElektraInitiative/PermaplanT/assets/53004139/f56be986-2163-4d39-8959-ab8f6138b199)
Inside test-and-build-mdbook during mdbook build.

Reply:
Great